timescaledb/tutorials/grafana/visualizations/candlestick.md

```diff
@@ -0,0 +1,167 @@
+# Build a candlestick chart in Grafana
+Candlestick charts show the opening, closing, high, and low prices 
+of financial assets, such as stocks, currencies, and securities. 
+They are mainly used in technical analysis, to predict how prices will change.
+
+They can answer questions like:
+
+*   What are the open, close, high, and low prices of an asset on this day?
+*   What is the spread between opening and closing prices over this time?
+*   How is the price of this asset changing over time?
+*   Is this asset entering bearish or bullish territory?
+
+   <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/candlestick/candlestick_fig.png" alt="Diagram of a candlestick. The open and close prices define the ending positions of a central box. Lines extend from the box to show the high and low prices. The distance between open and close prices is called the real body. The distance between the central box and the high price is the upper shadow, and the distance between the box and the low price is the lower shadow."/>
+
+The figure above shows the structure of a candlestick. A candlestick covers a
+specific time interval, for example 5 minutes, 10 minutes, or 1 hour. For this
+period, it plots four values:
+
+* **Open**: The starting price
+* **Close**: The closing price
+* **High**: The highest price
+* **Low**: The lowest price
+
+A candlestick chart can show many candlesticks over time. This helps you see
+patterns in the changing price of an asset. For example, you can tell whether an
+asset is entering bullish or bearish territory, or whether its market activity is
+topping or bottoming out.
+
+## What you will learn
+This tutorial shows you how to:
+
+*   [Create candlestick aggregates with raw data](#create-a-candlestick-with-raw-data)
+*   [Include volume when querying with raw data](#include-volume-when-querying-from-raw-data)
```

Comment:
Suggest renaming this because it's not entirely clear what the section is about from the heading alone. Maybe something like `Show transaction volume when querying with raw data`?

Reply:
done, thanks